shared/gh/js/views/gh.admin-batch-edit-date.js

```diff
@@ -86,45 +90,56 @@ define(['lodash', 'moment', 'gh.api.util', 'gh.api.config'], function(_, moment,
     /**
      * Add another day to the terms based on the selection of weeks
      *
+     * @param {Number}    [weekNumber]    The optional number of the week to add the events to
      * @private
      */
     var addAnotherDay = function() {
```

Comment:
I'm also wondering if we should do parameter validation in our view controllers?